liveman_api/lib/liveman_api.ex

```diff
@@ -1,18 +1,36 @@
 defmodule LivemanApi do
   @moduledoc """
-  Documentation for `LivemanApi`.
+  API Wrapper for Liveman
   """
 
-  @doc """
-  Hello world.
+  @timeout 30_000
 
-  ## Examples
+  def get(client, url, query \\ %{}),
```

Comment:
To be easier to use, `client` should not be a required param so that in the Livebook input a user can just write `LivemanApi.get(base_url, url, query)`

To achieve this, we can use Livebook inputs to set the base URL. Check out this video: https://youtu.be/MOTEgF-wIEI?t=446. These inputs would be at the top of the notebook so that we can change at will between the local/staging/production base URLs.

Then for the `LivemanAPI.get`, `base_url` would come from that input 💭

Reply:
Thanks, I've updated it to use an input field. It seems there's an option for URL, but `http://dockerhost:4000` isn't a valid URL 😂